src/client/src/rt-platforms/openfin/components/ToggleOpenFinLayoutLock.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { isOpenFin } from 'apps/MainRoute/widgets/contact-us/utils'
+import styled from 'styled-components/macro'
+import { toggleLockedLayout } from './utils'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faLock, faLockOpen } from '@fortawesome/free-solid-svg-icons'
+import { library } from '@fortawesome/fontawesome-svg-core'
+
+library.add(faLock, faLockOpen)
+
+const OpenFinOnlyButton = styled('button')`
+  ${isOpenFin ? '' : 'display: none;'}
```

Comment:
Is this necessary? This is in the OpenFin platform adapter, so presumably isn't rendered in other platforms?